src/test/kfs_rest_parser_test.cpp

```diff
@@ -864,6 +887,21 @@ TEST_F(KFSRestParserTest, parseInvalidRequestWithInputsMissing) {
     ASSERT_EQ(status, StatusCode::REST_NO_INPUTS_FOUND);
 }
 
+TEST_F(KFSRestParserTest, parseInvalidRequestStringInput_datatypeNotU8) {
+    std::string request = R"({
+    "inputs" : [
+        {
+        "name" : "input0",
+        "shape" : [ 3 ],
+        "datatype" : "UINT16",
+        "data" : [ "zebra", "openvino", "123"]
```

Comment:
Add test for multi-dimensional array or multiple types in data field, for example `"zebra", "abc", 12, "string"` or `12,56,89,"abc",100`